templates/index.html

```diff
@@ -18,55 +18,56 @@ <h3>Introduction</h3>
   <a href="https://github.com/ua-snap/data-api">GitHub</a>. Instructions on how
   to use each data endpoint is given below.
 </p>
+<!-- 
+The service categories are now defined in the application.py in the get_service_categories function. 
+They are alphebetized by category name and passed to this file.
```

Comment:
```suggestion
They are alphabetized by category name and passed to this file.
```
Thanks for leaving this comment here, I found it helpful 👍